e2e/wdio/core/pages/side-navigation.po.ts

```diff
@@ -0,0 +1,58 @@
+import { CoreBaseComponentPo } from './core-base-component.po';
+import { waitForElDisplayed, waitForPresent} from '../../driver/wdio';
+
+export class SideNavigationPo extends CoreBaseComponentPo {
+    private url = '/sideNavigation';
+
+    root = '#page-content';
+
+    NavigationExample = 'fd-side-navigation-example ';
+    TitilesExample = 'fd-side-navigation-titles-example ';
+    CompactExample = 'fd-side-navigation-compact-example ';
+    NonSelectableExample = 'fd-side-navigation-non-selectable-example ';
+    ThreeLevelsExample = 'fd-side-navigation-three-levels-example ';
+    MultipleSelectedExample = 'fd-side-navigation-multiple-selected-example ';
```

Comment:
for all of these, should start with lowercase letter
navigationExample
titlesExample
and so on...